crt_portal/cts_forms/templates/forms/complaint_view/resources/resource_table.html

```diff
@@ -94,10 +99,9 @@
                         {% with last=datum.resource.contact_last_name|default:"—" %}
                         {{ last|truncatechars:20 }}{% endwith %}, {% with first=datum.resource.contact_first_name|default:"—" %}
                         {{ first|truncatechars:20 }}
-                        {% endwith %}
-                        {{ datum.resource.contact_title }}
-                        {{ datum.resource.contact_affiliation }}
```

Comment:
was the removal of affiliation intentional?

(Didn't see it in the PR description, just checking)

Reply:
yep! I need to clarify with Amanda but I think the affiliation will just be the resource